src/components/NewCommentForm.tsx

```diff
@@ -75,25 +162,43 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
             placeholder="Type comment here"
-            className="textarea is-danger"
+            value={body}
+            onChange={event => {
+              setBody(event.target.value);
+              setBodyError(false);
+            }}
+            className={cn('textarea', {
+              'is-danger': bodyError,
+            })}
           />
         </div>
 
-        <p className="help is-danger" data-cy="ErrorMessage">
-          Enter some text
-        </p>
+        {bodyError
+          && (
+            <p className="help is-danger" data-cy="ErrorMessage">
+              Enter some text
+            </p>
+          )}
       </div>
 
       <div className="field is-grouped">
         <div className="control">
-          <button type="submit" className="button is-link is-loading">
+          <button
+            type="submit"
+            className={cn('button', 'is-link', {
+              kek: 'is-loading',
+            })}
+          >
             Add
           </button>
         </div>
 
         <div className="control">
           {/* eslint-disable-next-line react/button-has-type */}
-          <button type="reset" className="button is-link is-light">
+          <button
+            type="reset"
```

Comment:
it doesn't work - we have controlled fields
you should manually clear values (change state values)

also, note that it should clear errors according to the requirements